src/utils/Processing.ts

```diff
@@ -429,6 +429,19 @@ export async function init(): Promise<void> {
       configDoc.set(field, userValue);
     }
   });
+
+  // And for nested publisher fields
+  ['name', 'url'].forEach(field => {
+    const userValue = readlineSync.question(
+      `Publisher ${field.charAt(0).toUpperCase() + field.slice(1)} (Default: ${configDoc
```

Comment:
I'm not sure how much this matters, but I think you can alternatively use lodash's `upperFirst` function to do the same as this string concatenation.

Reply:
Good point, done in https://github.com/FHIR/sushi/pull/899/commits/b06aaf24c6f8cf172712947988bba9e8246ed902.